src/PrestaShopBundle/Resources/config/services/core/form/form_data_handler.yml

```diff
@@ -118,6 +118,13 @@ services:
     arguments:
       - '@prestashop.core.command_bus'
 
+  prestashop.core.form.identifiable_object.data_handler.order_return_form_data_handler:
+    class: 'PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler\OrderReturnFormDataHandler'
+    arguments:
+      - '@prestashop.core.command_bus'
+      - '@=service("prestashop.adapter.legacy.context").getContext().shop.id'
```

Comment:
Why injecting those two extra parameters? They are not used in the `OrderReturnFormDataHandler` class

Reply:
Yeah they shouldn't be needed. Will remove them!